packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -120,6 +145,9 @@ export class CeramicCliUtils {
       config.metrics.metricsExporterEnabled = process.env.CERAMIC_METRICS_EXPORTER_ENABLED == 'true'
     if (process.env.COLLECTOR_HOSTNAME)
       config.metrics.collectorHost = process.env.COLLECTOR_HOSTNAME
+    if (process.env.CERAMIC_NODE_DID_SEED) {
+      config.node.privateSeed = process.env.CERAMIC_NODE_PRIVATE_SEED_URL
```

Comment:
is this env var supposed to be different than the one in the enclosing if-statement? SEED vs SEED_URL

Reply:
thanks this was a typo